coldfront/core/allocation/views.py

```diff
@@ -414,6 +416,137 @@ def get_context_data(self, **kwargs):
 
         return context
 
+class AllocationListItem:
+    id: int
+    project_id: int
+    project_name: str
+    resource_name: str
+    department_number: str
+    allocation_status: str
+    pi_last_name: str
+    pi_first_name: str
+    pi_user_name: str
+    itsd_ticket: str
+    file_path: str
+    service_rate: str
+
+
+    def __init__(self, **kwargs):
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+
+class AllocationTableView(LoginRequiredMixin, ListView):
+
+    model = Allocation
+    template_name = 'allocation/allocation_table_view.html'
+    context_object_name = 'allocation_list'
+    paginate_by = 25
+
+    def get_queryset(self):
+
+        view_list: List[AllocationListItem] = []
+
+        allocation_search_form = AllocationSearchForm(self.request.GET)
+        
+
+        if not allocation_search_form.is_valid():
+            return view_list
+            data = allocation_search_form.cleaned_data
+            resource = Resource.objects.get(name="Storage2")
+            allocations = Allocation.objects.filter(resources=resource)
+
+            # Project Title
+            if data.get('project'):
+                allocations = allocations.filter(
+                    project__title__icontains=data.get('project'))
+
+            for allocation in allocations:
+                department_type = AllocationAttributeType.objects.get(name="department_number")
+                department_attribute = AllocationAttribute.objects.get(allocation=allocation, allocation_attribute_type=department_type)
```

Comment:
Note for other reviewers - we have a more efficient way to implement this section.  It's on my prototype branch for sorting/filtering and will be merged when that makes it to a PR.